Octal-Ideas/octal-educators
# creates a webhook for dialogflow and

connects it to our website using dialogflow messenger we have two working frontend chatbots
dialogflow is already connected to google api

# Description

This pull request fixes the issue where the chatbot in the frontend is consistently returning the same response regardless of the user input. The issue is resolved by implementing context-aware responses using Dialogflow Messenger, which is already connected to Google API.

The changes made include updating the chatbot's logic to analyze the user input and provide relevant responses based on the context, instead of relying on a default response. This ensures that the chatbot responds appropriately to different user inputs, improving its usability and effectiveness.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

The changes have been tested in the following manner:

1. Input various queries in the chatbot using different contexts
2. Verify that the chatbot returns relevant responses based on the input provided
3. Repeat the test with different user inputs to ensure consistent and accurate responses

**Test Configuration**:
* OS: Windows 10
* Browser: Google Chrome

## Checklist:
